src/c_api/c_api_function.cc

```diff
@@ -162,38 +162,35 @@ int MXCustomFunctionRecord(int num_inputs, NDArrayHandle *inputs,
                            MXCallbackList *callbacks) {
   using namespace mxnet;
   using namespace mxnet::custom_function;
-  using mxnet::autograd::AutogradRuntime;
```

Comment:
Is this function used only by custom ops registered in frontend?

Reply:
This is only used by autograd.Function